infra/gcb/build_project.py

```diff
@@ -276,7 +279,11 @@ def get_build_steps(project_dir):
               {
                   'name': 'gcr.io/oss-fuzz-base/base-runner',
                   'env': env,
-                  'args': ['bash', '-c', 'test_all'],
+                  'args': [
+                      'bash',
+                      '-c',
+                      'test_all || echo "check_build %s" && false' % failure_msg
```

Comment:
nit: use .format() for consistency with the rest?

Reply:
done!